disnake/guild.py

```diff
@@ -3311,6 +3312,89 @@ async def query_members(
             self, query=query, limit=limit, user_ids=user_ids, presences=presences, cache=cache
         )
 
+    async def get_or_fetch_members(
+        self,
+        member_ids: List[int],
```

Comment:
Maybe change this to `user_ids` to keep it consistent with `Guild.query_members` and the api docs.